### ISSUE TITLE
Enable IaC setup for PostgreSQL without GSA VPN

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/piipan#102]
In our IaC create-resources.bash script we use psql on the admin's workstation to connect to PostgreSQL, set up managed identities, etc. In order for this to work, we trust the GSA network, and we have to run the script will connected to that network.

Explore other mechanisms to avoid having to do this; for example [this ARM template that (temporarily) launches an Ubuntu VM within the database's virtual network](https://github.com/Azure/azure-postgresql/tree/master/arm-templates/ExampleWithStartupScript).

**Comment by cjhskippy** [comment]
Let's discuss.  Do we know enough about the FNS environment to be able to solve for their needs yet?

**Comment by ryanhofdotgov** [comment]
No, we don't have any info yet – no rush to do this, it can likely wait for several months.
